android/app/build.gradle

```diff
@@ -1,3 +1,14 @@
+plugins {
```

Comment:
The new flutter version changed the android gradle project structure a bit. These changes are only the ones related to firebase cherry picked from all changes for the flutter upgrade. I unfortunately can't test if the app can compile with just these changes. ☹️

Reply:
We should not merge this until we can be sure it does

Reply:
I hope #121 will help with this.

Reply:
The flutter version we use is too old to handle the new way of defining the plugins, so I have to revert this:

![CI failure](https://github.com/nt4f04uNd/sweyer/assets/6966049/ae5bcc79-7008-4946-85a9-d0a7cca6108d)